docs/docs/software-architecture.md

```diff
@@ -128,29 +121,20 @@ Backend:
 - Communication is JSON over HTTP
 - Docker / Kubernetes
 
-Orderbook:
-
-- Node.js express REST API
-- External API to get exchange rates
-
 Blockchain Gateway:
 
 - Bitcoin libraries:
-  - Bcoin (Zipkin, 2020)
   - Bitcoinjs (junderw, 2020)
 - Bitshare libraries:
   - Bitsharejs
-- To ensure Clean Architecture as the project might get expanded further, this will be serving as a mechanism to separate the blockchain logic from the orderbook:
-  - Responsible for issuing the creation of HTLCs on the requested blockchains
-  - Can also house the key generation and hashing modules as they are required for all chains
 
 HTLC/XXX:
```

Comment:
Might be that we use bip174 for the BTC htlc ("import { PsbtInput } from "bip174/src/lib/interfaces")